### PR TITLE
Flask-IIIF v0.1.0

### LINKED ISSUE #4: Release v0.1.0
- [ ] compile `RELEASE-NOTES.rst`, `CHANGES`
- [ ] update `flask_iiif/version.py`
- [ ] ...

### PR DESCRIPTION
**Pull request by jirikuncar**
* Initial public release.  (closes #4)

Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

--
cc @tiborsimko @drjova

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
-   :target: https://pypi.python.org/pypi/Flask-IIIF/
+    :target: https://pypi.python.org/pypi/Flask-IIIF/
 
 About
 =====
-Flask-IIIF is a Flask extension permitting to set up Shibboleth
-Single-Sign-On authentication in Flask based web applications.
+
+Flask-IIIF is a Flask extension permitting easy integration with the image
+IIIF API standards.
```
Comment (tiborsimko) [inline README.rst]: What about: 

- Flask-IIIF is a Flask extension permitting easy integration with the International Image Interoperability Framework (IIIF) API standards.

(Here and also in `setup.py` in the `description` that currently says "Flask-IIIF multimedia extension")

**Comment by tiborsimko** [comment]
`flask_iiif.cache.*` may be missing in the package?  Note `check-manifest`'s report:

```
missing from sdist:
  .gitmodules
  .travis.yml
  docs/_themes
  flask_iiif/cache
  flask_iiif/cache/__init__.py
  flask_iiif/cache/cache.py
  flask_iiif/cache/simple.py
suggested MANIFEST.in rules:
  include .travis.yml
  recursive-include flask_iiif *.py
```

File: README.rst (hunk 1 above)
Comment (jirikuncar) [inline README.rst]: done

**Comment by jirikuncar** [comment]
> ... `check-manifest` ...

done (cc @tiborsimko)

**Pull request merged**